HelpSource/Classes/FunctionDef.schelp

```diff
@@ -45,18 +45,30 @@ return a list of all references to a given symbol.
 
 method::argNames
 
-Get the Array of Symbols of the argument names.
+Get all of the arguments' names as a link::Classes/SymbolArray::.
 
 code::
 { |a = 9, b = 10, c| a + b }.def.argNames;
 ::
+
+method::argumentNamesForCall
+Same as link::#-argName::, but guarantees only the argument names the user sees are present.
```

Comment:
Same as link::#-argName::, but guarantees only the argument names the user sees are present.
–>
Returns the argument names in code (link::#-argNames:: may have `this` as first item).